package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "node your-script.js",
```

Comment:
- [ ] As requested by the previous code reviewer, please let's add a script to easily run the `dev server`, some like so: 
```
"start": "webpack serve --open",
```